src/schema/v2/lot.ts

```diff
@@ -24,3 +25,45 @@ export const Lot = new GraphQLObjectType<any, ResolverContext>({
     }
   },
 })
+
+export const Lot2 = new GraphQLObjectType({
+  name: "Lot2",
+  fields: () => ({
+    bidCount: {
+      type: GraphQLInt,
+    },
+    floorSellingPrice: {
+      type: Money,
+      resolve: resolveLotCentsFieldToMoney("floorSellingPriceCents"),
+    },
+    floorSellingPriceCents: {
+      type: GraphQLInt,
+    },
+    internalID: {
+      type: GraphQLString,
+    },
+    onlineAskingPrice: {
+      type: Money,
+      resolve: resolveLotCentsFieldToMoney("onlineAskingPriceCents"),
+    },
+    onlineAskingPriceCents: {
+      type: GraphQLInt,
+    },
+    reserveStatus: {
+      type: GraphQLString,
+    },
+    saleId: {
+      type: GraphQLString,
+    },
+    sellingPrice: {
+      type: Money,
+      resolve: resolveLotCentsFieldToMoney("sellingPriceCents"),
+    },
+    sellingPriceCents: {
+      type: GraphQLInt,
+    },
+    soldStatus: {
+      type: GraphQLString,
+    },
+  }),
+})
```

Comment:
Once bidding engine is moved to Gravity, this will be the type that we'll use. Correct @erikdstock? Thoughts? I remember you brought up some questions around naming; perhaps we can address that here.

Reply:
I don't think anything is final about future schema at this point... Because there are still some very open questions around websockets/graphql subscriptions among other things in part. My 0.02 would be that this work makes querying individual lot state from causality possible, as well as collections (that a user has watched or bidded upon), and this meets all needs except for live auctions (which have their own unique client implementations) - so this leaves us with a basically complete parity to the gravity-only implementation, and we will have liberty to select the most logical next step when that time comes.

Reply:
Updated to `CausalityLotState` while things are in flight.